### PR TITLE
Enable stylecheck linter and disable default exclude rules in favour of our own

### PR DESCRIPTION
**Pull request by webvictim**
WIP: fixing list of stylecheck complaints

### CODE REVIEW

File: lib/jwt/jwt.go [hunk 1]
```diff
@@ -220,7 +219,7 @@ func (k *Key) Verify(p VerifyParams) (*Claims, error) {
 	expectedClaims := josejwt.Expected{
 		Issuer:   k.config.ClusterName,
 		Subject:  p.Username,
-		Audience: jwt.Audience{p.URI},
+		Audience: josejwt.Audience{p.URI},
```
Comment (wadells) [inline lib/jwt/jwt.go:222]: Why is `josejwt` preferred over `jwt` here?  Looking at the imports it seems like it could have gone either way?

**Review by wadells — APPROVED** [review]

**Comment by awly** [comment]
@webvictim let us know when it's ready for review please

**Comment by webvictim** [comment]
Oh wow, necro PR. I'll take a look next week and see whether I can finish this off. It's draft because the tests started failing with some of these changes and I haven't had time to bisect what causes it yet.

**Comment by russjones** [comment]
@webvictim Should we close this PR? Have not seen movement on it for a while.

**Comment by webvictim** [comment]
Yes, let's close it off. I think it needs to be written by someone with more experience than me.

**Pull request closed without merging**